batchflow/models/torch/layers/conv_complex.py

```diff
@@ -3,6 +3,9 @@
 
 import torch
 from torch import nn
+import torch.nn.functional as F
+import torchvision.ops
+from torch.nn.modules.utils import _pair
```

Comment:
I am strongly against using functions from other packages' `utils`, especially ones with underscore in its name: those functions can be changed drastically, and this particular function saves you like 10 characters at most with loss at readability.
Anyhow, there is already a `to_n_tuple` function in `batchflow.models.torch.utils`.